src/k2/dto/Expression.cpp

```diff
@@ -0,0 +1,298 @@
+#include "Expression.h"
```

Comment:
Is it possible to move this out of dto? The code in this .cpp is only needed by the server, but I guess it would be weird to separate the .h and .cpp.